### PR TITLE
thrift: Suggest plugins over --yarpc

### PR DESCRIPTION
**Pull request by abhinav**

### CODE REVIEW

**Comment by prashantv** [comment]
Can we remove `--yarpc`?

**Comment by abhinav** [comment]
In a different release, yes. The plan is to release 0.2 with just plugins added, and 0.3 with `--yarpc` removed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ Releases
 v0.2.0 (2016-09-01)
 -------------------
 
--   Implemented a ThriftRW plugin. Check the documentation of the
+-   Implemented a ThriftRW plugin. This should now be used instead of the
+    ThriftRW `--yarpc` flag. Check the documentation of the
```
Comment (breerly) [inline CHANGELOG.md]: "Check the documentation of the" - ?

**Comment by breerly** [comment]
Since thriftrw-go gets installed globally, we should probably wait until all our customers are updated to remove `--yarpc`.

File: CHANGELOG.md (hunk 1 above)
Comment (abhinav) [inline CHANGELOG.md]: read the line below it

Comment (breerly) [inline CHANGELOG.md]: Ah, assumed that was another item because no highlighting 👍

**Comment by breerly** [comment]
lgtm

**Comment by prashantv** [comment]
Cool makes sense

**Pull request merged**